tests/unit/s2n_server_extensions_test.c

```diff
@@ -139,6 +156,88 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_connection_free(conn));
         }
 
+        /* Test TLS13 with Secure Negotiation */
```

Comment:
perhaps make the comment a little more detailed, eg. Test Secure Negotiation Server Hello Extension not sent with TLS 1.3